templates/apps/homebrew-settings.html

```diff
@@ -11,8 +11,8 @@
     </div>
     {{/each}}
 
-    <h2>Moo Man's House Rules</h2>
-    <p class="notes">Assorted mechanics change that I believe have potential to improve the game. Note that the ones listed here are not an exhaustive list of the house rules I use, but only those that can be applied in code.<br><br>For the entire list, please see the <a href="https://github.com/moo-man/WFRP4e-FoundryVTT/wiki/Homebrew-Rules">Wiki Page.</a><br>Note that these rules are generally all meant to be used together.<br>Feedback is always appreciated. Please use the <a href="https://github.com/moo-man/WFRP4e-FoundryVTT/discussions/categories/moo-s-house-rules">Github Discussion Board</a> for discussion of the mechanics.</p>
+    <h2>{{localize "SETTINGS.Moo"}}</h2>
+    <p class="notes">{{localize "SETTINGS.MooHint1"}}<br><br>{{localize "SETTINGS.MooHint2"}} <a href="https://github.com/moo-man/WFRP4e-FoundryVTT/wiki/Homebrew-Rules">Wiki Page.</a><br>{{localize "SETTINGS.MooHint3"}}<br>{{localize "SETTINGS.MooHint4"}} <a href="https://github.com/moo-man/WFRP4e-FoundryVTT/discussions/categories/moo-s-house-rules">Github Discussion Board</a> {{localize "SETTINGS.MooHint5"}}</p>
```

Comment:
In circumstances like this, a single localization entry would be preferable (even if the HTML tags go inside it as well)